### DISCUSSION TITLE
能从某个文件夹一键导入索引吗

### DISCUSSION THREAD

**Discussion by Amoscey** [tamlok/vnote#2134] [General]
有的时候从别的软件导出一个文件夹，还得手动导入。
这时候只能单个右键导入，文件多的时候就太麻烦了。

**Comment by tamlok** [comment]
可以全选右键导入，也可以先不导入，用到的时候，双击会自动导入并打开。后面会支持打开第三方文件夹。

**Reply by Amoscey** [reply]
为什么我这边多选文件之后右键点击没有菜单＞﹏＜现在只能单选文件，然后右键“导入到索引”导入……

**Reply by Amoscey** [reply]
哦哦我发现了，是不是超过多少个文件右键就没有菜单了，选择几个的话就还有。（感谢解答）

**Reply by tamlok** [reply]
外部文件和内部笔记不能混合选择
